src/components/cms/BotBuilder/BotBuilderPages/DesignViews/UIView.js

```diff
@@ -281,14 +278,40 @@ class UIView extends Component {
 
   handleChangeIconImage = event => {
     const file = this.botBuilderIcon.files[0];
-    const image = window.URL.createObjectURL(file);
     if (!event.target.files || !event.target.files[0]) {
       this.handleRemoveUrlIcon();
     } else {
-      this.setImageAvatar(image);
+      this.uploadAvatarImage(file, file.name);
     }
   };
 
+  uploadAvatarImage = (file, filename) => {
+    const { accessToken, actions } = this.props;
+    this.setState({ uploadingBotbuilderIconImg: true });
+    let form = new FormData();
+    form.append('access_token', accessToken);
+    form.append('image_name', filename);
+    form.append('image', file);
+    uploadImage(form)
+      .then(payload => {
+        this.setState({
+          uploadingBotbuilderIconImg: false,
+        });
+        actions.setBotAvatar({
+          botbuilderIconImg: `https://api.susi.ai/cms/getImage.png?image=${payload.imagePath}`,
```

Comment:
Do not hardcode the server url

Reply:
Can you use the getImage util here?

Reply:
@akshatnitd The irony here is yesterday only I made a PR for getImageUrl